docs/src/test/java/com/google/cloud/spring/autoconfigure/pubsub/it/PubSubTemplateDocumentationIntegrationTests.java

```diff
@@ -79,9 +79,12 @@ public void testCreatePublishPullNextAndDelete() {
 			//end::publish[]
 			PubsubMessage pubsubMessage = pubSubTemplate.pullNext(subscriptionName);
 
-			assertThat(pubsubMessage.getData()).isEqualTo(ByteString.copyFromUtf8("message"));
-			assertThat(pubsubMessage.getAttributesCount()).isEqualTo(1);
-			assertThat(pubsubMessage.getAttributesOrThrow("key1")).isEqualTo("val1");
+			await().atMost(Duration.TWO_SECONDS).untilAsserted(() -> {
```

Comment:
2 seconds is enough?

Reply:
Seemed to be. I can increase just to ensure it's not flaky